bin/create-test-db.php

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env php
```

Comment:
Have to be move inside the tests directory, we don't want to release this file.

Reply:
Fixed